razorpay-payments.php

```diff
@@ -102,7 +108,14 @@ function init_form_fields()
                         'authorize' => 'Authorize',
                         'capture'   => 'Authorize and Capture'
                     )
-                )
+                ),
+                'enable_webhook' => array(
+                    'title' => __('Enable Webhook', 'razorpay'),
+                    'type' => 'checkbox',
+                    'description' => esc_url( admin_url('admin-post.php') ) . '?action=rzp_webhook',
+                    'label' => __('Enable Razorpay Webhook with the URL listed below.', 'razorpay'),
+                    'default' => 'yes'
+                ),
```

Comment:
Add a `webhook_action` here that decides what we do with the order on payment success. There are multiple scenarios:

- Late Authorized (we get payment in authorized status, let the store owner decide)
- payment.failed (we can mark as failed) (very tricky, let us leave)
- Actual authorized, but post to woocommerce failed (we let store decide)

Store decisions:

1. Mark the payment as successful, if there is stock/inventory
2. Inform the store owner?